public/css/style.css

```diff
@@ -1,10 +1,19 @@
+body {
+    background-image: url("../assets/wood-background.png");
+}
+
 header, main{
     width: 80%;
 }
 main {
     margin-top: 40px;
 }
 
+.navbar {
+    background-color: #014421;
+    margin-top: 0;
```

Comment:
"navbar" is predefined class in bootstrap. Not ideal to customize unless you want all navbar in this app to be with this styling

Reply:
mt-0 class can be used to style margin-top: 0

Reply:
I specifically used navbar because I wanted the entire top bar to have thee same styling. Makes it easer for user-interaction.